homeassistant/components/sensor/sma.py

```diff
@@ -73,6 +76,9 @@ def _check_sensor_schema(conf):
     """Set up SMA WebConnect sensor."""
     import pysma
 
+    # Check config again during load - dependancy available
```

Comment:
```suggestion
    # Check config again during load - dependency available
```

Reply:
Thanks @tjorim ! I fixed this on Friday, but could not push due to issues on github. I pushed my fixes now

Reply:
This suggested change feature looks quite cool though!